odd-platform-specification/components.yaml

```
@@ -4063,6 +4063,18 @@ components:
         - GRAPH
         - ALL
 
+    GenAIRequest:
+      type: object
+      properties:
+        question:
+          type: string
+
+    GenAIQuestionResult:
+      type: object
+      properties:
+        question_response:
+          type: string
```

components and properties naming feels a little bit redundant
```suggestion
    GenAIRequest:
      type: object
      properties:
        body:
          type: string

    GenAIResponse:
      type: object
      properties:
        body:
          type: string
```

---

done